### ISSUE TITLE
Identify those districts/ pieces and civil Parishes that need to be duplicated in another county.

### ISSUE THREAD

**Issue by Captainkirkdawson** [FreeUKGen/FreeCENMigration#1043]
As we set up the new system we have a need to identify the pieces that have a parish(es) in a different county. As a Coordinator I know which pieces are the problem pieces in my County. But this is not necessarily always the case.
There will be a definite need to identify the relevant pieces. Short of going through every piece in England and Wales is there any method of identifying these pieces? I have tried to look at Districts down to civil parishes but as they are examined one by one and the county info has been stripped this is not possible.
Is there any method of generating a list of Parishes from the original data that was raked from TNA?
Alternatively can we run run a county / civil parish test against the Place Names Database and get a list of mis-matches?
These mis-matches will be the ones that Rhoda will need to transfer to a different county. If we have a list she will be able to work through them.
In short, we have the process but not the means of identifying the Pieces and Parishes to be transferred
Geoff

E2 U6 =S8  C5: P 21

**Comment by Captainkirkdawson** [comment]
@geoffj-FUG I was under the impression that the coordinator should know where boundary conditions exist. The extraction of the information you seem to be seeking is hard to envisage at this time.
We already run a test for a valid place name for district / piece and civil parish which is what formed the basis of story 
Complete linkages from Freecen2 Civil Parishes to Freecen2 Places #951
We currently have literally thousands of civil parishes that do not link to any Freecen2 Place as shown in that story.
This is why we use the piece place name if found and the district place name if it exists.
Even with this fall back we still have hundreds of Districts that need to be connected to an appropriate place. This issue is of far greater importance than what should be a few boundary conditions.

**Comment by PatReynolds** [comment]
Possibly use the work of the  FreeBMD District aliasing team?  @PatReynolds to ask the team

**Comment by PatReynolds** [comment]
Now business as usual, so closing.
